src/stores/GlobalStore.js

```diff
@@ -37,6 +42,24 @@ export default class GlobalStore {
     },
   });
 
+  setOnline = () => {
```

Comment:
this func should be under the constructor

Reply:
hmmm, I feel like I kinda understand why but also don't understand why.

Reply:
You mean in the constructor or outside of it but under it?

Reply:
just move the function below the constructor, but still in the Class.

class {
  class vars...
  computed vars...
  constructor()
  otherMethods()
}